metatron-app/metatron-discovery
# Notify users when they log in to another device

### Description
If users are already logged in on another device, users can choose whether or not to log in.

**Related Issue** : 

### How Has This Been Tested?
1. Login userA at Device1
2. Login userA at Device2
3. Check if the user is notified of IP duplication normally

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
